libs/npm/core/src/hooks/api-subscriber/interfaces/IColleagueModel.ts

```diff
@@ -0,0 +1,16 @@
+export interface IColleagueModel {
+  user:
+    | {
+        id: number | undefined;
+        username: string | undefined;
+        email: string | undefined;
+      }
+    | undefined;
+  colleague:
```

Comment:
Curious as to why we need an `IColleagueModel`  Aren't colleagues just users by another name?
When I hit the endpoint shouldnt it just return a list of colleagues? me as a user doesnt have to be repeated in every response?